From 4f122b0f545e9808a304a32d33ce41e0faa66f28 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 6 May 2019 21:22:11 -0600
Subject: [PATCH] [SQUASH][DNM] Merge tag 'android-9.0.0_r37' into
 staging/lineage-16.0_merge-android-9.0.0_r37

Android 9.0.0 Release 37 (PQ3A.190505.002)

* tag 'android-9.0.0_r37':
  hal: fix for voip volume setting when select_devices()

https://github.com/haggertk/android_hardware_qcom_audio/tree/lineage-16.0_merge-android-9.0.0_r37

Change-Id: I55c0ee0554c1a36602804564e36723fc4fe51095
---
 hal/audio_hw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hal/audio_hw.c b/hal/audio_hw.c
index 8e04bf8c8..f315b5e58 100644
--- a/hal/audio_hw.c
+++ b/hal/audio_hw.c
@@ -1500,7 +1500,7 @@ int select_devices(struct audio_device *adev,
             voice_set_sidetone(adev, out_snd_device, true);
     }
 
-    if (usecase == voip_usecase) {
+    if (usecase->type != PCM_CAPTURE && voip_usecase) {
         struct stream_out *voip_out = voip_usecase->stream.out;
         audio_extn_utils_send_app_type_gain(adev,
                                             voip_out->app_type_cfg.app_type,
-- 
2.17.1

